src/power_grid_model/utils.py

```diff
@@ -471,3 +472,45 @@ def _make_test_case(  # noqa: PLR0913
     params_json = json.dumps(params, indent=2)
     (output_path / "params.json").write_text(data=params_json, encoding="utf-8")
     (output_path / "params.json.license").write_text(data=LICENSE_TEXT, encoding="utf-8")
+
+
+def msgpack_deserialize_from_fileobj(
```

Comment:
thinking whether we should explicitly rename to `msgpack_deserialize_from_stream`. @figueroa1395 @nitbharambe do you have any opinion?